biosimulations/apps/dispatch-api/src/main.ts

```diff
@@ -12,12 +12,14 @@ import { CustomOrigin } from '@nestjs/common/interfaces/external/cors-options.in
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+  const globalPrefix = 'api';
```

Comment:
I dont think we want to add this back, correct?

Reply:
removed